Procfile

```diff
@@ -0,0 +1 @@
+web: gunicorn application:app --log-file -
```

Comment:
This is for deploying to Heroku. It's basically just a bash command that gets run to start the server.